g2p/app.py

```diff
@@ -1,20 +1,33 @@
 """
+The g2p Studio web app.
 
-Views and config to the g2p Studio web app
+You can run the app (and API) for development purposes on any platform with:
+    pip install uvicorn
+    uvicorn g2p.app:APP --reload --port 5000
```

Comment:
Trying that locally, and the locally spun up g2p-studio is all broken and generates tons of warnings at the console.
Was `pip install -e .[api]` and then this command the right thing to do?

Reply:
ah oh hmm. will have to review the documentation to make sure it makes sense

Reply:
Tested again, and I don't really see the problem I saw the first time. 
a) the warnings are on `origin/main` too, and are triggered by starting to type in the text box with the mapping set to Custom but no rules in there yet. I should make an issue of that.
b) this time I had done `pip install -e .[dev]` in this sandbox, and `uvicorn g2p.app:APP --reload --port 5000` seems to work just fine, so I'm not sure what happened when I first tested this.